.gitignore

```diff
@@ -1,3 +1,4 @@
 vim-markdown-preview.html
 /target
 **/*.rs.bk
+Cargo.lock
```

Comment:
Since this is a binary crate and not a library, my understanding is that Cargo.lock should be included in source control.